### ISSUE TITLE
Cannot Sit nor Break Chairs

### ISSUE THREAD

**Issue by 200Tigersbloxed** [efeketket/furnicraft#3]
When placing any sort of chair; such as a sofa, bench, chair, stool, etc., the chair cannot be sat in (at least by a player) and cannot be broken correctly. The bounding box is removed, but the chair model still stays visibly present, until the datapack is uninstalled.

![2024-03-26_09 07 05](https://github.com/efeketket/furnicraft/assets/45884377/6e4e7127-a490-4325-83c4-f609d660e47e)

![2024-03-26_09 07 18](https://github.com/efeketket/furnicraft/assets/45884377/ec09c268-c88a-409e-acbc-c63f2b32503a)

Currently using Fabric 1.20.1 with the [Data loader](https://modrinth.com/mod/dataloader) mod to load the datapack into all worlds. I have noticed this also happens regardless of using a mod loader or not, and whether or not you are using a server or client (multiplayer/singleplayer).

Let me know if I can provide any more information!

**Comment by 200Tigersbloxed** [comment]
After doing more research, 1.20.2 is also affected. The reason is most likely due to a missing tag.

```hack
[Worker-Main-20/ERROR]: Couldn't load tag ketket_furnitures:cantsit as it is missing following references: minecraft:wind_charge (from file/furnicraft v2.25)
```

This is output in the logs on both 1.20.1 and 1.20.2, but not on 1.20.3 and 1.20.4 (the two versions that work fine).

**Comment by 200Tigersbloxed** [comment]
Sure enough, removing line 35 from [`cantsit.json`](https://github.com/efeketket/furnicraft/blob/main/FurniCraft/data/ketket_furnitures/tags/entity_types/cantsit.json#L35) fixes the issue for 1.20.1 and 1.20.2!

Here is the edited `cantsit.json` for a quick fix.
```json
{
    "replace": false,
    "values": [
        "minecraft:interaction",
        "minecraft:item",
        "minecraft:item_display",
        "minecraft:block_display",
        "minecraft:armor_stand",
        "minecraft:item_frame",
        "minecraft:player",
        "minecraft:experience_orb",
        "minecraft:arrow",
        "minecraft:area_effect_cloud",
        "minecraft:dragon_fireball",
        "minecraft:end_crystal",
        "minecraft:ender_pearl",
        "minecraft:evoker_fangs",
        "minecraft:eye_of_ender",
        "minecraft:falling_block",
        "minecraft:fireball",
        "minecraft:firework_rocket",
        "minecraft:glow_item_frame",
        "minecraft:leash_knot",
        "minecraft:lightning_bolt",
        "minecraft:llama_spit",
        "minecraft:marker",
        "minecraft:painting",
        "minecraft:small_fireball",
        "minecraft:text_display",
        "minecraft:tnt",
        "minecraft:potion",
        "minecraft:snowball",
        "minecraft:wither_skull",
        "minecraft:experience_bottle",
        "minecraft:fishing_bobber",
        "minecraft:egg",
        "minecraft:shulker_bullet",
        "minecraft:trident",
        "#minecraft:arrows"
    ]
}
```

**Comment by efeketket** [comment]
Thank you for taking the time to fix this! I missed the "minecraft:wind_charge" tag that was added in 1.20.2. I'll fix it.
